client/state/data-layer/wpcom/concierge/schedules/shifts/schema.json

```diff
@@ -8,7 +8,7 @@
 			"end_timestamp"
 		],
 		"properties": {
-			"id": { "type": "string" },
+			"id": { "type": "integer" },
```

Comment:
Since we actually won't be using this `id` field in the future, I'm going to remove it here.